nestkernel/parameter.h

```diff
@@ -1724,6 +1724,275 @@ class DimensionParameter : public Parameter
 };
 
 
+/**
+ * Parameter class representing an exponential distribution applied on a parameter.
+ */
+class ExpDistParameter : public Parameter
+{
+public:
+  using Parameter::value;
+
+  /**
+   * Construct the parameter from a dictionary of arguments.
+   */
+  ExpDistParameter( const DictionaryDatum& d );
+
+  /**
+   * Copy constructor.
+   */
+  ExpDistParameter( const ExpDistParameter& p )
+    : Parameter( p )
+    , p_( p.p_->clone() )
+    , beta_( p.beta_ )
+  {
+    parameter_is_spatial_ = true;
+  }
+
+  ~ExpDistParameter() override
+  {
+    delete p_;
+  }
+
+  /**
+   * @returns the value of the parameter.
+   */
+  double
+  value( RngPtr, Node* ) override
+  {
+    throw BadParameterValue( "Exponential distribution parameter can only be used when connecting." );
```

Comment:
Is this documented somewhere? It should probably be stated in the docstring for the exponential distribution. The same holds for all the new ones.